release_notes/ocp-4-8-release-notes.adoc

```diff
@@ -579,6 +579,21 @@ The bundle format is the preferred Operator packaging format for Operator Lifecy
 
 For more information, see xref:../operators/operator_sdk/osdk-pkgman-to-bundle.adoc#osdk-pkgman-to-bundle[Migrating package manifest projects to bundle format].
 
+[discrete]
+[id="ocp-4-8-builds"]
+=== Builds
+
+// https://bugzilla.redhat.com/show_bug.cgi?id=1969963
+[id="ocp-4-8-builds-telemetry-metric"]
+==== New Telemetry metric for number of builds by strategy
+Telemetry includes a new `openshift:build_by_strategy:sum` gauge metric, which sends the number of builds by strategy type to the Telemeter Client. This metric gives site reliability engineers (SREs) and product managers visibility into the kinds of builds that run on {product-title} clusters.
```

Comment:
Have we ever posted release notes for things we send to Telemeter?

/cc @simonpasquier

Reply:
https://docs.openshift.com/container-platform/4.7/release_notes/ocp-4-7-release-notes.html contains two release notes related to "telemetry".

Reply:
I don't think we have a strict rule about mentioning telemetry metrics in the changelog. I don't think it hurts to include the new metric but it would be nice to be consistent across components. FWIW the [documentation](https://docs.openshift.com/container-platform/4.7/support/remote_health_monitoring/about-remote-health-monitoring.html) links to the cluster-monitoring-operator repository which is the source of truth to know which metrics are collected by the telemetry service.
@sichvoge and @brampling for their opinion.

Reply:
We're open about which metrics we collect so I don't see any reason not to mention them in the release notes. I do agree that we should try to be consistent though, and include all or none.

Reply:
> FWIW the [documentation](https://docs.openshift.com/container-platform/4.7/support/remote_health_monitoring/about-remote-health-monitoring.html) links to the cluster-monitoring-operator repository which is the source of truth to know which metrics are collected by the telemetry service.

@simonpasquier  I've updated the "Information collected by Telemetry" topic you mentioned with this specific item: https://github.com/openshift/openshift-docs/pull/34032

Reply:
@adambkaplan @simonpasquier @brampling Okay - so we're good to go with this release note.